src/test/java/org/folio/circulation/domain/RequestTypeItemStatusWhiteListTests.java

```diff
@@ -0,0 +1,135 @@
+package org.folio.circulation.domain;
+
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+import static org.folio.circulation.domain.ItemStatus.WITHDRAWN;
+import static org.folio.circulation.domain.RequestTypeItemStatusWhiteList.canCreateRequestForItem;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+@RunWith(JUnitParamsRunner.class)
+public class RequestTypeItemStatusWhiteListTests {
+
+  @Test
+  public void canCreateHoldRequestWhenItemStatusCheckedOut() {
+    assertTrue(canCreateRequestForItem(ItemStatus.CHECKED_OUT, RequestType.HOLD));
```

Comment:
Personally, I might statically import the `ItemStatus` and `RequestType` to reduce the noise in these statements